### ISSUE TITLE
[Regression] RPCS3 search for game files not only in game folder but also in parent folders (see last comment)

### ISSUE THREAD

**Issue by SleepyTails** [RPCS3/rpcs3#11089]
Game:
[BLUS30833] Blades of Time

Issue:
Game does not boot. No intro, just black screen. FPS counter also stops.
In log: 
F VM: Access violation writing location 0x0 (unmapped memory)  [type=u32] 

Build where it works:
rpcs3-v0.0.18-12916-962e140c_win64

Build where regression first found:
rpcs3-v0.0.18-12946-5e436984_win64

System:
Microsoft Windows [Version 10.0.19041.630]
Intel(R) Core(TM) i7-5700HQ CPU @ 2.70GHz | 4 core 8 Threads
GPU: 'GeForce GTX 965M' 445.87

Image and logs:

![build12966](https://user-images.githubusercontent.com/93467108/139574410-20880052-8e0d-4c9a-a6ae-ab6d8321da85.png)

[RPCS3.log.regression.zip](https://github.com/RPCS3/rpcs3/files/7448350/RPCS3.log.regression.zip)
[RPCS3.log.workingbuild.zip](https://github.com/RPCS3/rpcs3/files/7448352/RPCS3.log.workingbuild.zip)

**Comment by SleepyTails** [comment]
Well, I've tried this on other machine and it works.

Then I found the reason
My game location is:
D:\Games\PS3\new checked\_todo_Blades of Time\BLUS30833
And inside folder D:\Games\PS3 I have a file PS3_DISC.SFB
Do not know why this file was here and what it is for, but this file location is not in game folder. If I remove this file -> game starts without any issues, so it looks like emulator is searching for files not only in game folder but in every parent folder.
Also some other games boots on new version of rpcs3 even with this file.

For me it still looks like bug (because on 12916 it booted normally with this file), so I leave ticket open
